test/mock_query_test.dart

```diff
@@ -1232,10 +1359,13 @@ void main() {
         'name': 'Springfield',
         'state': 'Massachusetts',
       },
+      {
+        'name': 'Springfield',
+        'state': 'Missouri',
+      },
     ]);
 
-    snapshots = await baseQuery.endAt(['Springfield', 'Missouri']).get();
-
+    snapshots = await baseQuery.endAt(['Wellington']).get();
```

Comment:
Should it return everything because Wellington is alphabetically grater than Washington, or because "Wellington" as a value does not exist, and so it returns everything?

Reply:
If it's the latter, it'd be nice to add a comment to explain it, and perhaps change Wellington to some dummy value so that it is obvious why the query returns those results.

Reply:
Wellington is alphabetically greater so it returns everything that is under it. Do you want me to rename wellington to something clearer ? Do you have a variable name in mind ?

Reply:
No, Wellington is fine. I just wasn't sure what the expected behavior was. Since edge cases with queries can get complicated, perhaps you can add a comment to explain that if the value is greater than all others, we expect everything to be returned. Thank you!